AerofoilX/GpSystemServices_X.cpp

```diff
@@ -8,6 +8,7 @@
 #include <pthread.h>
 #else
 #include "SDL2/SDL.h"
+#include <pthread.h>
```

Comment:
If this is non-conditional then please just move it out of the #ifdef

Reply:
This has been updated.  What does AerofoilX refer to?  Unix?

Reply:
Unix-like OS basically